rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/PluginDescriptor.java

```diff
@@ -111,7 +111,7 @@ public boolean isDynamic() {
 
     }
 
-    PluginDescriptor(String pluginName, String pluginDescription) {
+    public PluginDescriptor(String pluginName, String pluginDescription) {
         this.pluginDescription = pluginDescription;
         this.pluginName = pluginName;
         this.configurableFields = new ArrayList<Field>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)